public/pages/Transforms/containers/Transforms/EditTransform.test.tsx

```diff
@@ -0,0 +1,126 @@
+        </CoreServicesContext.Provider>
+      </Router>
+    ),
+  };
+}
+
+describe("<EditTransform /> spec", () => {
+  it("renders the component", async () => {
+    browserServicesMock.transformService.getTransform = jest.fn().mockResolvedValue({
+      ok: true,
+      response: testTransform,
+    });
+    const { container } = renderEditTransformWithRouter([`${ROUTES.EDIT_TRANSFORM}?id=${testTransform._id}`]);
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it("sets breadcrumbs when mounting", async () => {
+    browserServicesMock.transformService.getTransform = jest.fn().mockResolvedValue({
+      ok: true,
+      response: testTransform,
+    });
+    renderEditTransformWithRouter([`${ROUTES.EDIT_TRANSFORM}?id=${testTransform._id}`]);
+
+    expect(coreServicesMock.chrome.setBreadcrumbs).toHaveBeenCalledTimes(1);
+    expect(coreServicesMock.chrome.setBreadcrumbs).toHaveBeenCalledWith([
+      BREADCRUMBS.INDEX_MANAGEMENT,
+      BREADCRUMBS.TRANSFORMS,
+      BREADCRUMBS.EDIT_TRANSFORM,
+      { text: testTransform._id },
+    ]);
+  });
+
+  it("adds error toaster when get transform has error", async () => {
+    browserServicesMock.transformService.getTransform = jest.fn().mockResolvedValue({ ok: false, error: "some error" });
+    const { getByText } = renderEditTransformWithRouter([`${ROUTES.EDIT_TRANSFORM}?id=${testTransform._id}`]);
+
+    await waitFor(() => {});
+
+    expect(coreServicesMock.notifications.toasts.addDanger).toHaveBeenCalledTimes(1);
+    expect(coreServicesMock.notifications.toasts.addDanger).toHaveBeenCalledWith("Could not load transform job test1: some error");
+
+    await waitFor(() => getByText("Testing transform edit page"));
+  });
+
+  it("adds error toaster when get transform throws error", async () => {
+    browserServicesMock.transformService.getTransform = jest.fn().mockRejectedValue(new Error("rejected error"));
+    const { getByText } = renderEditTransformWithRouter([`${ROUTES.EDIT_TRANSFORM}?id=${testTransform._id}`]);
+
+    await waitFor(() => {});
+
+    expect(coreServicesMock.notifications.toasts.addDanger).toHaveBeenCalledTimes(1);
+    expect(coreServicesMock.notifications.toasts.addDanger).toHaveBeenCalledWith("rejected error");
+    await waitFor(() => getByText("Testing transform edit page"));
+  });
+
+  it("can edit description", async () => {
+    browserServicesMock.transformService.getTransform = jest.fn().mockResolvedValue({
+      ok: true,
+      response: testTransform,
+    });
+
+    browserServicesMock.transformService.putTransform = jest.fn().mockResolvedValue({
+      ok: true,
+      response: testTransform,
+    });
+    const { getByTestId } = renderEditTransformWithRouter([`${ROUTES.EDIT_TRANSFORM}?id=${testTransform._id}`]);
+
+    await waitFor(() => {});
+
+    fireEvent.focus(getByTestId("description"));
+    await userEvent.type(getByTestId("description"), "some description");
+    fireEvent.blur(getByTestId("description"));
```

Comment:
Is there supposed to be an assert somewhere here to confirm it was updated to some description?